latch/registry/table.py

```diff
@@ -302,6 +317,10 @@ def _obj_field(k: str, x: JsonValue) -> l.ObjectFieldNode:
 
 def _json_value(x: JsonValue) -> l.ValueNode:
     # note: this does not support enums
+
+    if isinstance(x, l.Node):
```

Comment:
change the parameter type annotation to be `Union[JsonValue, l.Node]`?

Reply:
oh yeah that's actually not enough anyway since I need it at any level of the tree